src/Constants/index.js

```diff
@@ -183,3 +183,5 @@ export const ApprovalStatus = {
   Rejected: "rejected",
   Submitted: "submitted",
 };
+
+export const allowedUserRoles = [UserType.Admin, UserType.Operator];
```

Comment:
Might be overkill for this small thing, but something to look into for future enhancements is to use the RBAC config and hook we use in other apps for controlling who has access to what